islandora.module

```diff
@@ -957,7 +957,7 @@ function islandora_islandora_object_purged($pid) {
   $cache_meta = CacheableMetadata::createFromRenderArray([
     '#cache' => [
       'tags' => [
-        $pid,
+        "islandora_object:$pid",
```

Comment:
Almost feel like we should just make [`IslandoraFedoraObject::getDrupalCacheObjectTag()`](https://github.com/discoverygarden/islandora/blob/bc4d12b12625d6cfba691d60b488a2a377828b84/src/Tuque/IslandoraFedoraObject.php#L48-L59) public, and then just call it.